src/components/CustomFileInput/index.js

```diff
@@ -36,10 +36,39 @@ export default class CustomFileInput extends Component<Props> {
     event.currentTarget.value = '' // Allow re-uplading the same file
   }
 
+  handleDragOverEvent = (ev: DragEvent) => {
+    // Prevent default behavior (Prevent file from being opened)
+    ev.preventDefault();
+  }
+
+  getFileFromDragEvent = (ev: SyntheticDragEvent<HTMLSpanElement>): ?File => {
+    let dataTransfer: DataTransfer = ev.dataTransfer;
+    let items: DataTransferItemList = dataTransfer.items;
+    let files: FileList = dataTransfer.files;
+
+    if (items) {
+      let file: ?File = items[0].getAsFile();
+      return file;
+    } else if (files) {
+      let file: ?File = ev.dataTransfer.files[0].name;
```

Comment:
You can just use the variable `files` that you declared earlier
`let file: ?File = files[0].name;`